types/riscript/index.d.ts

```diff
@@ -0,0 +1,54 @@
+declare class RiScript {
```

Comment:
The upstream package is also a CJS module; should this have proper exports and an import in the test, rather than declaring a global?